src/mil_actions.tex

```diff
@@ -39,7 +39,7 @@
 \newlength{\fhArmiesFleets} \setlength\fhArmiesFleets{8\baselineskip}
 \newlength{\frameArmiesFleetsY} \setlength\frameArmiesFleetsY{\calc{\textheight - \fhArmiesFleets}}
 
-\newlength{\fhNavalBridge} \setlength\fhNavalBridge{11\baselineskip}
+\newlength{\fhNavalBridge} \setlength\fhNavalBridge{12\baselineskip}
```

Comment:
Strange that you needed to do this. When I generated the pdf then the second bullet point spanned only 2 lines in that box. Perhaps we have different luatex versions. Which one do you have? Mine says this when I request its version in terminal:
```
$ luatex --version
This is LuaTeX, Version 1.14.0 (TeX Live 2022/dev/Debian)
```

I'd merge it in regardless as it looks fine both ways, but then the next time I fix something I'd need to check that box again just in case

Reply:
I have 1.0.4, the latest I get upgrading on Ubuntu Bionic